### PR TITLE
Example of reviewing code

### PR DESCRIPTION
**Pull request by Artufelii**
Example of how I review a code

### CODE REVIEW

**Review by Artufelii — COMMENTED** [review]
Maybe you should use semantic tags to improve the SEO of your website

**Review by didierganthier — CHANGES_REQUESTED** [review]
Hi @Artufelii ,

🎉 Good job so far! You're making great progress on this project, but there are still a few areas that need attention before we can move on to the next step.



## Highlights
- Used GitHub review feature ✔️ 

## Required Changes 🛠
- [ ] Well done on using the GitHub Review feature but you should  write at least 2 comments in the review.
  - You should use the [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) at least once to leave review comments.
  - Make sure you leave the comment under a line or block of code that fits the context the comment is about

- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- [ ] Nice review but you could use a couple of emojis and/or gifs in the review.
   * They make the code review more human-friendly 👽 🤖 

- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code.

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @Artufelii  :wave: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some changes you still need to work on before you move on to the next project but you are almost there.



 ## Highlights :+1: 
:heavy_check_mark: Good commit history.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Your project is well documented


 ## Required Changes ♻️
_Check the comments under the review._

- [ ] As requested by the previous reviewer, Please make sure that you perform a review using the GitHub Review feature and add at least 2 comments to the review. You need to use the [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) at least once to leave review comments. Make sure your comments are politely stated and give a reason for your suggestions.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Artufelii — COMMENTED** [review]
Excellent work doing this project 👌👌, you are almost done with the requirement, but there are some changes that you have to make!

File: index.html [hunk 1]
```diff
@@ -3,16 +3,16 @@
...
 <body>
 <main id="main">
 	<div id="title-container">
 		<h1 id="title"> Lionel Andres Messi Cuccittini </h1> 
-		<h3> The new king after Maradona </h3>
+		<p> The new king after Maradona </p>
 	</div>
 	<div id= "img-div"> 
-		<img id="image" src="./messi.jpeg" alt=""> 
+		<img id="image" src="./messi.jpeg">
```
Comment (Artufelii) [inline index.html:15]: Please add an alt property for every img tag on your document to improve your SEO

File: index.html [hunk 2]
```diff
@@ -3,16 +3,16 @@
 <head>
 	<link rel="stylesheet" href="styles.css">
 	<meta charset="UTF-8">
-	<title>Microverse trials</title>
+	<title>Microverse trials review</title>
 </head>
 <body>
 <main id="main">
 	<div id="title-container">
```
Comment (Artufelii) [inline index.html:10]: Excellent work follow the instructions but, kindly ask you to use semantic tags for every component of your code, which will help other developers to understand better your work.

**Review by Artufelii — COMMENTED** [review]
Excellent work following the instructions for this project 👌👌, you are almost done with this part, but there are some changes to make to accomplish your goal.

**Review by nahnahdev — APPROVED** [review]
## Project Status: APPROVED :green_circle: 
Hi @Artufelii , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- Good use of inline comments :heavy_check_mark: 
- Good job adding why a change was requested :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**